tests/globalkilltest/global_kill_test.go

```diff
@@ -111,6 +131,77 @@ func (s *TestGlobalKillSuite) connectPD() (cli *clientv3.Client, err error) {
 	return cli, nil
 }
 
+func (s *TestGlobalKillSuite) startTiKV(dataDir string) (err error) {
+	s.tikvProc = exec.Command(*tikvBinaryPath,
+		fmt.Sprintf("--pd=%s", *pdClientPath),
+		fmt.Sprintf("--data-dir=tikv-%s", dataDir),
+		"--addr=0.0.0.0:20160",
+		"--log-file=tikv.log",
+		"--advertise-addr=127.0.0.1:20160",
+	)
+	log.Infof("starting tikv: %v", s.tikvProc)
+	err = s.tikvProc.Start()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	go func() {
+		err := s.tikvProc.Wait()
+		log.Info("tikv exited: ", err)
+	}()
```

Comment:
Is this goroutine necessary? I have some question about this goroutine.
1. Do this will cause data race in Wait()?
2. The check in L582 and the check in cleanCluster() is not enough?

Reply:
1. it won't cause data race
2. it is meant for knowing if the pd or tikv exited abnormally during development, but since it is done, I'll remove it to make it lean.